apps/api/src/bootstrap.ts

```diff
@@ -121,7 +122,8 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
   await setupSwagger(app);
 
   Logger.log('BOOTSTRAPPED SUCCESSFULLY');
-
+  app.useGlobalFilters(new AllExceptionsFilter(app.get(PinoLogger)));
+  Logger.log('Filter Applied');
```

Comment:
Can we move this comment into the constructor of the exception filter? I guess it's instantiated once on bootstrapping. That way we will have everything contained.

Reply:
removed